brewery-app/src/components/Result.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+
+const Div = styled.div`
```

Comment:
same comment as above, unique names for each of these if possible

Reply:
will move to styles folder